egs2/urdu_shrutilipi/asr1/local/data.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Could you please follow the existing recipe data.sh for checking db.sh for where to save data?